test/spec/features/popup-menu/PopupMenuSpec.js

```diff
@@ -260,37 +304,14 @@ describe('features/popup', function() {
     }));
 
 
-    it('should open menu for specific element', inject(function(popupMenu) {
-
-      // when
-      popupMenu.registerProvider('menu', menuProvider);
-
-      popupMenu.open({}, 'menu', { x: 100, y: 100 });
-
-      var currentProvider = popupMenu._current.provider;
-
-      // then
-      expect(currentProvider.getHeaderEntries()).to.deep.include({
-        id: 'entry1',
-        label: 'foo'
-      });
-
-      expect(currentProvider.getEntries()).to.deep.include({
-        id: 'entry2',
-        label: 'foo'
-      });
-
-    }));
-
-
     it('should throw error when no provider', inject(function(popupMenu) {
 
       // when not registering a provider
 
       // then
       expect(function() {
         popupMenu.open({}, 'foo', { x: 100, y: 100 });
-      }).to.throw('Provider is not registered: foo');
```

Comment:
Why don't we assert the error message anymore?

Reply:
No specific reason. I'll update it.